test/delegates_test.rb

```diff
@@ -2,56 +2,78 @@
 require 'minitest/autorun'
 require_relative '../delegates'
 
+# Mocking HOMEPAGE_IMAGE_RIGHTS_HASH constant to return rights for a specific image_id
+class CustomDelegate
+  HOMEPAGE_IMAGE_RIGHTS_HASH = {
```

Comment:
Just calling out here that this "mocking" method is risky.

If this file gets required, the actual rights hash in the file will be overridden with these mocks. So we must be sure that the production code never actually requires this test file. It doesn't.